src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
 # Fill in database info
 # Example <path-to-database> = localhost:8889/planopedia
-spring.datasource.url=jdbc:mysql://<path-to-database>
-spring.datasource.username=<username>
-spring.datasource.password=<password>
```

Comment:
Any thoughts about adding this to .gitignore and let folks roll their own values?  probably don't need this coming in on pull requests.